### PR TITLE
Feature/composer with robo

### PR DESCRIPTION
**Pull request by dbosen**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -51,6 +55,17 @@
         ]
     },
     "scripts": {
+        "dependency-cleanup": [
+            "rm -rf bin",
```
Comment (dbosen) [inline composer.json]: Please add this cleanup to the script handler class instead of executing the rm directly in the composer json.

Comment (tjwelde) [inline composer.json]: Thank you for your input.
It is now solved.

**Comment by tjwelde** [comment]
The user has to set a hash_salt, before trying to install the site via robo. If he doesn't set it, the installation will fail on updating the locale. If he then fixes it and tries to install again, the script won't run, because the site is already installed.

Solutions:
1. Set empty hash_salt in local settings.php and add `die('Please provide a hash_salt')`
2. Set hash_salt in local settings.php by calculating one
3. If site is already installed, still do the locale update, config export and database dump

**Comment by tjwelde** [comment]
Regarding issues with config not exporting to correct path:

The problem is, that the Drush finder script (launched from Acquia DevDesktop) can't find the locally (with the site) installed drush, because we moved the bin folder out of the vendor directory. It then uses Drush from Acquia DevDesktop which is too old and shouldn't be relied on, since we already install the correct drush version for our site.

Solutions:
1. Provide a .drush-use file, which contains the path to the drush.launcher
2. Provide a drush wrapper, which contains the path and additional options

**Comment by tjwelde** [comment]
The drush wrapper didn't work (It could be, that it only uses local commands specific to the site), but the first option did.
Please test again

**Comment by tjwelde** [comment]
nice

**Pull request merged**